lisa/combinators/grid_combinator.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Got it from reading all lines, but I think future readers deserve docs on both base and specialization.

Reply:
Document will be added by Wenyi. I will add some code comments.